ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -164,10 +168,18 @@ public SafeFuture<HeaderWithFallbackData> builderGetHeader(
                         .map(ExecutionPayloadWithValue::getValue)
                         .orElse(UInt256.ZERO);
                 logReceivedBuilderBid(signedBuilderBid.getMessage());
-                if (signedBuilderBid.getMessage().getValue().lessOrEqualThan(localPayloadValue)) {
+
+                if (isLocalPayloadValueWinning(signedBuilderBid, localPayloadValue)) {
+                  LOG.info(
+                      "Using local Execution Payload instead of Builder Bid as it's challenged. "
+                          + "Builder bid value: {}, local block value: {}, builderBidChallengePercentage: {}%",
```

Comment:
`The local execution payload value ({}) was awarded the block over the builder payload ({}), {}% challenge configured.`